app/src/ingest_edd_web.py

```diff
@@ -0,0 +1,57 @@
+import json
+import logging
+import sys
+
+from smart_open import open as smart_open
+
+from src.adapters import db
+from src.app_config import app_config
+from src.db.models.document import Chunk, Document
+from src.util.ingest_utils import process_and_ingest_sys_args
+
+logger = logging.getLogger(__name__)
+
+
+def _ingest_edd_web(
+    db_session: db.Session,
+    json_filepath: str,
+    doc_attribs: dict[str, str],
+) -> None:
+    with smart_open(json_filepath, "r", encoding="utf-8") as json_file:
+        json_items = json.load(json_file)
+
+    urls_processed: set[str] = set()
+    for item in json_items:
+        if item["url"] in urls_processed:
+            # Workaround for duplicate items from web scraping
+            logger.warning("Skipping duplicate URL: %s", item["url"])
```

Comment:
Will timebox investigating a better way to dedup in Scrapy but this workaround works for now.

Reply:
Some duplicates are not detected b/c the default `fingerprint()` checks the Request's URL, not a resolved one, so the following URLs are considered different:
```python
from scrapy.utils.request import fingerprint, Request

url="https://edd.ca.gov/en/jobs_and_training/WDSFP_Workforce_Development_Solicitations_for_Proposals/"
print(fingerprint(Request(url)))
url2="https://edd.ca.gov/jobs_and_training/WDSFP_Workforce_Development_Solicitations_for_Proposals/"
print(fingerprint(Request(url2)))
url3="https://edd.ca.gov/jobs_and_training/WDSFP_Workforce_Development_Solicitations_for_Proposals.htm"
print(fingerprint(Request(url3)))
url4="https://www.edd.ca.gov/jobs_and_training/WDSFP_Workforce_Development_Solicitations_for_Proposals.htm"
print(fingerprint(Request(url4)))
```

Output:
```
b'!\x81\x045\xe2>\xc6\x85\xfc^k\xbe\xd9 \x96\x17rs\x9c\xa9'
b'\xc7\xa4\xa7]b$\x99_\r&\x89\xad\xa8\x88\x18\x01\xa6\x82\xde\x01'
b'^\xd45\xb7.\xd71\xbbi\xa1\xeb\xc7d[,H\xbd\xb4\xc1<'
b'm\xf9\xdbt\x00\xe2\x82\xee^\xae\x87\xfb}R\x8aBaof\xd5'
```